extension.ts

```diff
@@ -179,7 +179,7 @@ export async function activate(context: vscode.ExtensionContext) {
     showCmdLine("", modeHandlerToEditorIdentity[new EditorIdentity(vscode.window.activeTextEditor).toString()]);
   });
 
-  'rfbducw['.split('').forEach(key => {
+  'rfbducw[ax'.split('').forEach(key => {
```

Comment:
hah! there's no order, but yeah, this thing is pretty dumb. honestly i think we should parse them directly out of package.json and DRY this up, but that's a job for another day.